src/fragments/lib/analytics/js/autotrack/page-tracking.mdx

```diff
@@ -0,0 +1,120 @@
+## Page View Tracking
+
+If you want to track which page/url in your webapp is the most frequently viewed one, you can use this feature. It will automatically send events containing url information when the page is visited.
+
+To turn it on:
+
+```javascript
+import { configureAutoTrack } from 'aws-amplify/analytics';
+
+configureAutoTrack({
+  // REQUIRED, turn on/off the auto tracking
+  enable: true,
+  // REQUIRED, the event type, it's one of 'event', 'pageView' or 'session'
+  type: 'pageView',
+  // OPTIONAL, the customizable options for the event type.
+  options: {
+    // OPTIONAL, the attributes of the event
+    attributes: {
+      customizableField: 'attr'
+    },
+
+    // OPTIONAL, the event name, by default is 'pageView'
+    eventName: '',
+
+    // OPTIONAL, by default is 'multiPageApp'
+    // you need to change it to 'singlePage' if your app is a single-page app like React
+    appType: 'multiPageApp',
+
+    // OPTIONAL, provide the URL for the event.
+    urlProvider:  () => {
+      // the default function
+      return window.location.origin + window.location.pathname;
+    }
+  }
+});
+```
+
+You can turn it off by:
+```javascript
+import { configureAutoTrack } from 'aws-amplify/analytics';
+
+configureAutoTrack({
+  enable: false,
+  type: 'pageView'
+});
+```
+
+## Page Event Tracking
+
+If you want to track user interactions with elements on the page, you can use this feature. All you need to do is attach the specified selectors to your dom element and turn on the auto tracking.
+
+To turn it on:
+
+```javascript
+import { configureAutoTrack } from 'aws-amplify/analytics';
+
+configureAutoTrack({
+  // REQUIRED, turn on/off the auto tracking
+  enable: true,
+  // REQUIRED, the event type, it's one of 'event', 'pageView' or 'session'
+  type: 'event',
+  // OPTIONAL, the customizable options for the event type.
+  options: {
+    // OPTIONAL, the attributes of the event
+    attributes: {
+      customizableField: 'attr'
+    },
+    // OPTIONAL, events you want to track, by default is 'click'
+    events: ['click'],
+
+    // OPTIONAL, the prefix of the selectors, by default is 'data-amplify-analytics-'
+    // in order to avoid collision with the user agent, according to https://www.w3schools.com/tags/att_global_data.asp
+    // always put 'data' as the first prefix
```

Comment:
Nit:
```suggestion
    // Per https://www.w3schools.com/tags/att_global_data.asp, always start the
    // the prefix with 'data' to avoid collisions with the user agent
```